tests/test_dynamic_publisher_block.py

```diff
@@ -152,3 +152,23 @@ def test_closing(self):
             event.wait(.3)
 
             pub.return_value.close.assert_called_once()
+
+    @patch(DynamicPublisher.__module__ + '.Publisher')
+    def test_never_expiring(self, publisher):
+        block = DynamicPublisher()
+        topic = "topic.{{ $sig }}"
+
+        self.configure_block(block, dict(
+            topic=topic,
+            ttl=dict(seconds=-1),
+        ))
+
+        block.start()
+        publisher.assert_not_called()
```

Comment:
I think this (`assert_not_called`) is another one that is a newer assertion.

Alternative:
```python
self.assertEqual(publisher.call_count, 0)
```